Resource.rdef

```diff
@@ -10,10 +10,7 @@ resource app_version {
 	internal = 0,
 
 	short_info = "Mobile phone manager",
-	long_info = #'CSTR' array {
-		"This program helps to manage contacts stored on a mobile phone a"
-		"nd send/receive SMS messages."
-	}
+	long_info = "This program helps to manage contacts stored on a mobile phone and send/receive SMS messages."
```

Comment:
Even for a long_info this is a bit long...
How about: "Manage contacts on a mobile phone and send/receive SMS."